docs/source/user_guide/01a_installation.rst

```diff
@@ -0,0 +1,34 @@
+
+Installation
```

Comment:
Hmm. I am fine having installation be in the docs, but then we should probably remove it from the README and link to the version in the docs -- otherwise they are bound to get out of sync.